.github/actions/synthetics-stack-e2e-run/create-cluster.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+kibana_docker_image=${1:?}
+stack_version=${2:?}
+
+create_output_file="${RUNNER_TEMP}/cluster.json"
+
+oblt-cli cluster create custom \
+  --template="ess" \
+  --cluster-name-prefix="synthetics-stack-e2e" \
+  --parameters="{\"StackVersion\":\"${stack_version}\", \"KibanaDockerImage\": \"${kibana_docker_image}\", \"EphemeralCluster\": true}" \
```

Comment:
There are some missing parameters that are populated automatically when using the oblt-cli GitHub actions. https://github.com/elastic/apm-pipeline-library/blob/99241aabc666143859edd306cc231cd55b22b850/.github/actions/oblt-cli-create-custom/action.yml#L42-L69

I wonder whether https://github.com/elastic/apm-pipeline-library/blob/main/.github/actions/oblt-cli/action.yml could be changed to support `wait` in addition to the returning the cluster name